rfc/0007-community-team.md

```diff
@@ -1,3 +1,12 @@
+# Amendments
```

Comment:
I'm not a fan of how you have done this kind of amending. This might work fine for the first amendment, but what about two or three years down the line?

I recommend either entirely superseding the RFC (with changes marked), or editing RFC 7 directly and letting version control sort out the history for everyone.

Reply:
Agreed. That was a large part of the motivation for doing these in a git repo in the first place.

Reply:
We had no amendment stuff (that I could see) in the RFC process RFC, so I had no idea what was expected really - it'd be simple enough to do that, if that's the right path.

Reply:
Agreed, the git diff should be enough for history. Viewing the current version in its entirety is the most important thing after all.

Reply:
I agree, i think the amendments format makes it a bit harder to follow what the current state is. git can take care of showing changes

Reply:
This has been done.